CPAC/pipeline/schema.py

```diff
@@ -98,6 +98,12 @@
                 str, {'components': int, 'method': str}
             ),
         },
+    },
+    'amplitude_low_frequency_fluctuation': {
+        'target_space': ['Native', 'Template']
+    },
+    'regional_homogeneity': {
+        'target_space': ['Native', 'Template']
```

Comment:
Since these are the same, we could just have a single `target_space` instead of one for each of these https://github.com/FCP-INDI/C-PAC/blob/e81170fffbedfd11a1c04c0f402a1ad052e923c2/CPAC/pipeline/schema.py#L995 
https://github.com/FCP-INDI/C-PAC/blob/e81170fffbedfd11a1c04c0f402a1ad052e923c2/CPAC/pipeline/schema.py#L1013 like
```Python
'target_space': [In(valid_options['target_space'])],
```
for both of those lines and
```suggestion
    'target_space': ['Native', 'Template'],
```
here.

Reply:
Oh I see, yeah I like this idea. I can make these changes and test it before the merge